.github/scripts/test_db_sync.sh

```diff
@@ -38,18 +42,32 @@ else
     DB_PATH="chains/testnet/"
     TARGET_CHAIN="wss://ws.test.azero.dev"
 fi
-if [[ "${PRUNING}" == "true" ]]; then
-    DB_ARG="--enable-pruning"
+
+DB_ARG=""
+if [[ "${PRUNING}" == "true" || "${PARITY_DB}" == "true" ]]; then
+    if [[ "${PRUNING}" == "true" ]]; then
```

Comment:
This is now addressed

Reply:
Only partially, this `if` structure now makes no sense, it should just be setting the parameters separately based purely on the relevant variable, i.e.
```
# the rocksdb case
if [[ "${PARITY_DB}" == "true" ]]; then
 # the second case below, overwriting the first one
fi
if [[ "${PRUNING}" == "true" ]]; then
 # the first case below, preferably appending, see below
fi
```
preferably with the latter just appending to the variables – this is almost doable completely naturally now, except the `.html` would have to be removed from the `..FILE_NAME` variable (and perhaps rename this to just `LATEST_SNAPSHOT_NAME`, since there is not actual file with the `.html` extension, it's just the URI.

Reply:
You're right, now reworked